main.go

```diff
@@ -99,8 +102,21 @@ func realMain(echoChan chan<- *echo.Echo) int {
 		return 5
 	}
 
+	// Plugin Manager
+	homeDir, _ := os.UserHomeDir()
+	pluginExePath, _ := os.Executable()
```

Comment:
Nice to have would be if we could override it using Environment variables (for. e.g. in some docker images we keep everything in the `/app` folder i.e. `/app/logproxy , /app/plugins/rtrFilter`

Reply:
Added in https://github.com/philips-software/logproxy/pull/64/commits/740cd3dc226126cf1895a407ae57cb46b53923d7